coffee_httpd/Cargo.toml

```
@@ -6,7 +6,10 @@ edition = "2021"
 [dependencies]
 actix-web = "4"
 clap = "4.1.11"
+tokio = "1.22.0"
```

```suggestion
tokio = "1.22.0"
```

I think this is still wrong, you should brink inside only the feature of tokio that brink the `sync` package, take a look to the tokio rust docs, it is a well documented package

---

I only included the `sync` module from tokio